crawler

```diff
@@ -0,0 +1,15 @@
+This is just basic crawler,we can simply put in the url at the place of sample url
+import requests
+from bs4 import BeautifulSoup
+def web(page,WebUrl):
```

Comment:
There should be two blank lines before defining the function. Refer: https://www.python.org/dev/peps/pep-0008/#blank-lines